llama-index-integrations/readers/llama-index-readers-telegram/llama_index/readers/telegram/base.py

```diff
@@ -56,6 +58,8 @@ def load_data(
         entity_name: str,
         post_id: Union[int, None] = None,
         limit: Union[int, None] = None,
+        start_date: Union[datetime.date, None] = None,
```

Comment:
should those 2 date parameter be optional?

Reply:
Yeah, if you leave them None it will use the same logic as before this PR. So it'll load the last n messages/posts in the channel/group. But if you want to load a messages from specific period of time, for example from 01.01.2024 to 01.02.2024, you'll need to pass them dates in the loader. That's why I made them optional parameters

Reply:
oh I mean like `start_date: Optional[datetime.date]`

Reply:
Ahh, I see, so it'll be like `start_date: Optional[datetime.date] = None`. And same for the `end_date`

Reply:
Okay, I'll push the updates tomorrow as I am not at the desk right now 👍